templates/dashboard/order/pdf/invoice.html

```diff
@@ -5,39 +5,43 @@
 
 <html>
 <head>
-  <title>Invoice, order ID: {{ group.order.id }}</title>
-  <link rel="stylesheet" type="text/css" href="{% static '/assets/document.css' %}" />
+  <title>Invoice, order ID: {{ order.id }}</title>
```

Comment:
Missing translation

Reply:
Also "order ID: 3123" does not look like something we'd like users to see.

Reply:
Changed to:
title:
"Invoice for Order #123"
header:
"Order #123"

Reply:
👍